packages/components/carousel/src/use-carousel.ts

```diff
@@ -255,6 +255,14 @@ export const useCarousel = (
       }
     }
   )
+
+  const exposeActiveIndex = computed(() => {
+    if (isItemsTwoLength.value) {
+      return activeIndex.value % 2
+    }
+    return activeIndex.value
+  })
```

Comment:
```suggestion
  const exposeActiveIndex = computed({
    get: () => {
      if (isItemsTwoLength.value) {
        return activeIndex.value % 2
      }
      return activeIndex.value
    },
    set: (value) => {
      activeIndex.value
    }
```
The current activeIndex seems to support assignment, so we should keep the behavior consistent.